api/src/main/java/org/openmrs/module/eptsreports/reporting/library/cohorts/EriDSDCohortQueries.java

```diff
@@ -771,16 +756,14 @@ public CohortDefinition getN3() {
     CohortDefinition patientsEnrolledOnGaac = getAllPatientsEnrolledOnGaac();
     CohortDefinition startOrContinueGAAC = getPatientsWithStartOrContinueGAAC();
     CohortDefinition completedGAAC = getPatientsWhoCompletedGAAC();
-    CohortDefinition txCurr = txCurrCohortQueries.getTxCurrCompositionCohort(cohortName, true);
 
     String mappings = "onOrBefore=${endDate},location=${location}";
-    cd.addSearch("TxCurr", EptsReportUtils.map(txCurr, mappings));
     cd.addSearch("patientsEnrolledOnGaac", mapStraightThrough(patientsEnrolledOnGaac));
     cd.addSearch("startOrContinueGAAC", EptsReportUtils.map(startOrContinueGAAC, mappings));
     cd.addSearch("completedGAAC", EptsReportUtils.map(completedGAAC, mappings));
 
     cd.setCompositionString(
-        "TxCurr AND (patientsEnrolledOnGaac OR startOrContinueGAAC) NOT (completedGAAC)");
+        "(patientsEnrolledOnGaac OR startOrContinueGAAC) NOT (completedGAAC OR PregnantAndBreastfeedingAndOnTBTreatment)");
```

Comment:
PregnantAndBreastfeedingAndOnTBTreatment is missing from the .addSearch

Reply:
It should be removed (#180).